docs/products/kafka/howto/set-kafka-parameters.rst

```diff
@@ -0,0 +1,44 @@
+Manage Apache Kafka parameters
+===================================
+
+Aiven for Apache Kafka comes by default with a preset of configuration parameters defining for example automatic topic creation, partition count, replication factor, and retention time. These service configurations can be retrieved and managed via the Aiven web console, under the *Advanced configuration* section of the *Overview*
```

Comment:
I'm thinking how to make this paragraph lighter

Newly created **Aiven for Apache Kafka** service comes with a set of configuration properties that define values for partition count, replication factor, retention time, whether an automatic topic creation is enabled or not and others. 

You can read and edit  these configuration properties in the *Advanced configuration* section of the *Overview* page in Aiven web console or using the :ref:`Aiven CLI service update command <avn-cli-service-update>`.

Reply:
this is the summary so you don't need to specify "using the ``-c`` options" since we have this information again below